packages/terra-dynamic-grid/src/styles.js

```diff
@@ -0,0 +1,156 @@
+// IE < 11 doesn't honor window.CSS.Supports. Edge has supports, but will fail on grid.
+
+const gridTemplateColumns = (layout) => {
+  const columns = layout['grid-template-columns'];
+  if (columns === undefined) {
+    return {};
+  }
+
+  // If 2 values are specified, column is the 2nd value, else its the first
+  const gap = (layout['grid-gap'] || '0').split(' ').slice(-1)[0];
+  const msColumns = columns.split(/\s+/).join(` ${gap} `);
+
+  return {
+    'grid-template-columns': columns,
+    '-ms-grid-columns': msColumns,
+  };
+};
+
+
+const gridTemplateRows = (layout) => {
+  const rows = layout['grid-template-rows'];
+  if (rows === undefined) {
+    return {};
+  }
+
+  // If 2 values are specified, row is the 1st value
+  const gap = (layout['grid-gap'] || '0').split(' ')[0];
+  const msRows = rows.split(/\s+/).join(` ${gap} `);
+
+  return {
+    'grid-template-rows': rows,
+    '-ms-grid-rows': msRows,
+  };
+};
+
+
+const gridColumnStart = (region) => {
+  const start = region['grid-column-start'];
+  if (start === undefined) {
+    return {};
+  }
+
+  const msStart = (start * 2) - 1;
+
+  return {
+    'grid-column-start': `${start}`,
+    '-ms-grid-column': `${msStart}`,
+  };
+};
+
+const gridColumnEnd = (region) => {
+  if (region['grid-column-end'] === undefined
+    && region['grid-column-end'] === undefined) {
+    return {};
+  }
+
+  const start = region['grid-column-start'] || 0;
```

Comment:
actually, thinking about this... it may be possilbe to just not render this attribute if `grid-coloumn-end` is not specified.